source/_docs/scripts/conditions.markdown

```diff
@@ -383,28 +383,18 @@ Note that if only `before` key is used, the condition will be `true` _from midni
 [sun_trigger]: /docs/automation/trigger/#sun-trigger
 
 <div class='note warning'>
-The sunset/sunrise conditions do not work in locations inside the polar circles, and also not in locations with a highly skewed local time zone.
-
-In those cases it is advised to use conditions evaluating the solar elevation instead of the before/after sunset/sunrise conditions.
+The sunset/sunrise conditions do not work in locations inside the polar circles, and also not in locations with a highly skewed local time zone. In those cases it is advised to use conditions evaluating the solar elevation instead of the before/after sunset/sunrise conditions.
 </div>
 
 This is an example of 1 hour offset before sunset:
+
 ```yaml
 condition:
   condition: sun
   after: sunset
   after_offset: "-01:00:00"
 ```
 
-This is 'when dark' - equivalent to a state condition on `sun.sun` of `below_horizon`:
-
-```yaml
-condition:
-  - condition: sun
-    after: sunset
-    before: sunrise
-```
```

Comment:
This not functioning is a bug, not a documentation error.

Reply:
Not according to the documented principle that all conditions must be true within the same 24-hour period. There is no way you can be before sunrise AND after sunset on the same day.

I agree it would be nicer, but it could complicate the interpretation of multiple conditions.

Reply:
Well, it may be a bug, but it does not mean we should document what does not work....

https://github.com/home-assistant/core/issues/70781

Reply:
We don't document references to bugs in our documentation. Please revert these changes

Reply:
Updated to reflect intent. Suggeste fix here: https://github.com/home-assistant/core/issues/70781#issuecomment-1201481041

Reply:
Also, I find "not documenting references to bugs" rather silly.

Documentation is most useful when it reflects what is, not what should be.

Reply:
> Documentation is most useful when it reflects what is, not what should be.

That is true, the reality is though, that workarounds and temporary things to work with issue tent to stay in the documentation as well. Which makes it all really hard to maintain as well. Hence, we don't document bugs.